normandy/recipes/tests/test_serializers.py

```diff
@@ -15,6 +15,7 @@ def test_it_works(self, rf):
         assert serializer.data == {
             'name': recipe.name,
             'id': recipe.id,
+            'last_updated': Whatever(),
```

Comment:
@mythmon is there a better way to fix this test? If I try to set it to something like `timezone.now()` I get a failure where `last_updated` doesn't match up: 

```
{'last_updated': '2016-06-09T19:32:08.115291Z'} != {'last_updated': datetime.datetime(2016, 6, 9, 19, 32, 8, 120002, tzinfo=<UTC>)}
```